src/app/server/Server.cpp

```diff
@@ -282,7 +282,9 @@ void InitServer(AppDelegate * delegate)
     }
     else
     {
+#if CHIP_DEVICE_CONFIG_ENABLE_PAIRING_AUTOSTART
```

Comment:
nit: move #if before else statement.

```
#if ...
else
{
...
}
#endif
```

Reply:
Having if statement without else is not MISRA compliant and I noted that in community empty else blocks are started to appear

Reply:
Ok we can leave it, though as you mentioned the code is full of such code.